metadata/src/test/java/com/redhat/lightblue/metadata/mongo/MongoMetadataTest.java

```diff
@@ -173,6 +173,25 @@ public void createMdTest() throws Exception {
         Assert.assertEquals("testEntity", names[0]);
     }
 
+    @Test
+    public void createMdWithAndRefTest() throws Exception {
+        Extensions<JsonNode> extensions = new Extensions<>();
+        extensions.addDefaultExtensions();
+        extensions.registerDataStoreParser("mongo", new MongoDataStoreParser<JsonNode>());
+        JSONMetadataParser parser = new JSONMetadataParser(extensions, new DefaultTypes(), new JsonNodeFactory(true));
+
+        // get JsonNode representing metadata
+        JsonNode jsonMetadata = AbstractJsonNodeTest.loadJsonNode(getClass().getSimpleName() + "-qps-andquery.json");
```

Comment:
Test file missing.  I'll merge this anyway with this test ignored so builds are not broken.  It's blocking work @derek63 is doing for hystrix.